packages/wallets/localize-loader.js

```diff
@@ -6,6 +6,13 @@ import generateKey from './src/utils/generate-keys';
 
 const messages = new Map();
 const values = new Set();
+let defaultMessages;
+if (fs.existsSync('./src/translations/en.json')) {
+    const entries = Object.entries(JSON.parse(fs.readFileSync('./src/translations/en.json', 'utf-8')));
+    defaultMessages = new Map(entries);
+} else {
+    defaultMessages = new Map();
```

Comment:
if the file does not exist - should we explicitly throw an error? to make sure it _**does**_ exist

Reply:
In this case I don't think we should throw an error, the build command will always run `update-translations.sh` which will always download the Crowdin translation files (en.json, id.json, fr.json...)
In the case where the translation files are not available, the worst case is that the strings will not be translated, which is valid because there are not translations for that specific key